docs/Get-MarkdownMetadata.md

```diff
@@ -78,60 +75,55 @@ external help file platyPS-help.xml
 schema             2.0.0
 ```
 
-Retrives metadata from all markdown file at the directory path provided.
+This command gets metadata from each of the markdown files in the .\docs folder.
 ## PARAMETERS
 
 ### -Path
-Path to markdown file or folder.
-Markdown files typically use extension .md
+Specifies an array of paths of markdown files or folders.
 
 
 ```yaml
 Type: String[]
 Parameter Sets: FromPath
-Aliases: 
+Aliases:
 
 Required: True
 Position: Named
-Default value: 
+Default value:
 Accept pipeline input: True (ByPropertyName, ByValue)
 Accept wildcard characters: False
 ```
 
 ### -Markdown
-String object containing markdown.
+Specifies a string that contains markdown formatted text.
 
 
 ```yaml
 Type: String
 Parameter Sets: FromMarkdownString
-Aliases: 
+Aliases:
 
 Required: True
 Position: Named
-Default value: 
+Default value:
 Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
 ### CommonParameters
-This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```

Comment:
this change would be lost on the next `Update-MarkdownHelp`, but we can update the template string
https://github.com/PowerShell/platyPS/blob/master/src/Markdown.MAML/Resources/MarkdownStrings.resx#L175